common/persistence/pinot/pinotVisibilityStore_test.go

```diff
@@ -60,6 +73,286 @@ var (
 	}
 )
 
+type PinotVisibilitySuite struct {
```

Comment:
Let's write these tests in table test format as discussed and shown. No need to use suite. The indirection and redundancy in suites is something we want to avoid and instead have more declarative table style tests. 
You already came up with test cases (suite functions) so converting them to table should be straightforward. Let me know if you need any help